src/app/globals.css

```diff
@@ -108,21 +108,35 @@ form label {
 }
 
 #my-gui-container-compare-1 {
-  width: 200px;
```

Comment:
not a super big fan of fixed size and using pixels for widths but ill fix it in a future pr